src/OrchardCore/OrchardCore.Data.YesSql/SqliteHelper.cs

```diff
@@ -6,18 +6,27 @@ namespace OrchardCore.Data;
 
 public static class SqliteHelper
 {
-    public static string GetConnectionString(SqliteOptions sqliteOptions, ShellOptions shellOptions, string shellName) =>
-        GetConnectionString(sqliteOptions, GetDatabaseFolder(shellOptions, shellName));
+    public static string GetConnectionString(SqliteOptions sqliteOptions, ShellOptions shellOptions, string shellName, string databaseName = "")
+        => GetConnectionString(sqliteOptions, GetDatabaseFolder(shellOptions, shellName), databaseName);
 
-    public static string GetConnectionString(SqliteOptions sqliteOptions, string databaseFolder) =>
-        new SqliteConnectionStringBuilder
+    public static string GetConnectionString(SqliteOptions sqliteOptions, string databaseFolder, string databaseName = "")
+    {
+        if (string.IsNullOrEmpty(databaseName))
         {
-            DataSource = Path.Combine(databaseFolder, "yessql.db"),
+            // For backward compatibility, we assume that the database name is 'yessql.db'.
+            databaseName = "yessql.db";
```

Comment:
Could you please use constants for old and new database names

Reply:
why? it's only used in one place.